yellowbrick/regressor/residuals.py

```diff
@@ -153,15 +166,15 @@ def score(self, X, y=None, **kwargs):
         -------
         score : float
         """
-        self.score_ =  self.estimator.score(X, y, **kwargs)
+        # super will set score_ on the visualizer
+        super(PredictionError, self).score(X, y, **kwargs)
```

Comment:
Will be nice to have this for #925 and with your updates to `RegressionScoreVisualizer`